composer.json

```diff
@@ -35,7 +35,7 @@
     },
     "extra": {
         "branch-alias": {
-            "dev-master": "2.x-dev"
+            "dev-master": "3.x-dev"
```

Comment:
if we want only to support php8.4, we don't need to change version

Reply:
I did not make this change, so no idea about this. It looks like I branched from master, so this was already there. You should correct it yourself to be correct.

Reply:
you need to rebase on `2.x`